### PR TITLE
feat(cli): authentication and token revocation

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
Added scripts for authentication and token revocation.

### CODE REVIEW

**Review by Johaney-s — COMMENTED** [review]
this commit should be marked as breaking change, because it requires adding new property to config

**Review by cuadradek — COMMENTED** [review]
Looks good, and I tested it - it works. I have only one suggestion - please, add _help_ method and batch mode to both scripts - just like in script _removeOidcTokens_.

**Comment by cuadradek** [comment]
> Looks good, and I tested it - it works. I have only one suggestion - please, add _help_ method and batch mode to both scripts - just like in script _removeOidcTokens_.

We discussed this with @Johaney-s and batch mode for _oidcAuthentication_ script is not needed, because the authentication link would be printed anyway (when batch mode is on, nothing is printed but we have to print the authentication link anyway).

File: perun-cli/oidcAuthentication [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+use strict;
+use warnings;
+
+use Perun::auth::OidcAuth;
+use Perun::Exception;
+use Getopt::Long qw(:config no_ignore_case);
+
+sub help {
+	return qq{
+	OIDC authentication
```
Comment (Johaney-s) [inline perun-cli/oidcAuthentication:12]:
```suggestion
	Performs OIDC authentication which stores new access and refresh tokens. Can be used to renew tokens before performing batch operations.
```

**Review by Johaney-s — APPROVED** [review]

**Review by cuadradek — APPROVED** [review]

**Pull request merged**